### PR TITLE
there can be only vmem!

### PR DESCRIPTION
**Pull request by kilobyte**
😼

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/vmem/1)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,59 @@
...
-## Version Conventions
-
-- **Builds** are tagged something like `0.2+b1`, which means _Build 1 on top of version 0.2_
-- **Release Candidates** have a '-rc{version}' tag, e.g. `0.2-rc3, meaning _Release Candidate 3 for version 0.2_
-- **Stable Releases** use a _major.minor_ tag like `0.2`
 
 ## Pre-Built Packages for Windows
 
-The recommended and easiest way to install PMDK on Windows is to use Microsoft vcpkg. Vcpkg is an open source tool and ecosystem created for library management.
+The recommended and easiest way to install VMEM on Windows is to use Microsoft vcpkg. Vcpkg is an open source tool and ecosystem created for library management.
```
Comment (marcinslusarz) [inline README.md]: There's no VMEM in vcpkg yet.

File: README.md [hunk 2]
```diff
@@ -283,7 +234,7 @@ run different types of tests.
 
 ### Memory Management Tools
 
-The PMDK libraries support standard Valgrind DRD, Helgrind and Memcheck, as well as a PM-aware version of [Valgrind](https://github.com/pmem/valgrind) (not yet available for FreeBSD). By default, support for all tools is enabled. If you wish to disable it, supply the compiler with **VG_\<TOOL\>_ENABLED** flag set to 0, for example:
+The VMDK libraries support standard Valgrind DRD, Helgrind and Memcheck, as well as a PM-aware version of [Valgrind](https://github.com/pmem/valgrind) (not yet available for FreeBSD). By default, support for all tools is enabled. If you wish to disable it, supply the compiler with **VG_\<TOOL\>_ENABLED** flag set to 0, for example:
```
Comment (marcinslusarz) [inline README.md]: VMDK?

**Comment by marcinslusarz** [comment]
- VERSION shouldn't exist
- the merge of src/Makefile looks suspicious - ALL_TARGETS contains examples, where previously it didn't, RELEASE_TARGETS - the same
- src/Makefile - pkg-cfg-common generates rasdeps variable
- src/common/.cstyleignore - what's up with pmemcompat.h change?
- src/test/Makefile - envconfig changes seem to be wrong
- src/test/Makefile.inc - uuid.o is commented out, should be removed?, vpath is commented out
- src/test/RUNTESTS - pmemdetect and pmem_fs_dir support is commented out
- src/test/vmem_malloc/TEST2 - that's cheating
- utils/build-dpkg.sh - $PACKAGE_NAME should probably be changed

We need separate Docker images for this repo.

File: README.md (hunk 1 above)
Comment (kilobyte) [reply README.md]: I've marked that "yet".

File: README.md (hunk 2 above)
Comment (kilobyte) [reply README.md]: ✓

**Comment by kilobyte** [comment]
VERSION: ✓.
ALL_TARGETS: ✓.
rasdeps: ✓.
.cstyleignore for pmemcompat.h: same in PMDK, cstyle can't handle templates in this file.
envconfig: please elaborate.
uuid.o, vpath: ✓.
pmemdetect: let's discuss: what's the point of separate pmem/nonpmem tests?
cheating: will need to find devdax some way.
PACKAGE_NAME: only place that sets it, says "vmem".  That's ok?
Docker: let's trick someone else into contributing ☺

**Comment by marcinslusarz** [comment]
src/Makefile - what about RELEASE_TARGETS?
envconfig - %s without argument
src/test/Makefile.inc - $(TOP)/src/nondebug/common/util_posix.o**_**
pmemdetect - I'm not saying "we should have such tests", I'm saying "don't leave commented out code"
cheating - ok, I'm waiting
PACKAGE_NAME - yeah, I missed this change
Docker - it's trivial, but I can do this after the merge
src/common/common.rc - VMDK again?

**Comment by kilobyte** [comment]
RELEASE_TARGETS: unused, removed.
envconfig: was used only for ndctl libs, removed.
util_.posix.o**_**: fixed
VMDK: not in my tree, which was in sync with Github.

**Comment by marcinslusarz** [comment]
```
-                       VALUE "ProductName", "Persistent Memory Development Kit"
+                       VALUE "ProductName", "Volatile Memory Development Kit"
```

**Review by marcinslusarz — APPROVED** [review]

**Pull request merged**